.github/workflows/verify.yml

```diff
@@ -0,0 +1,69 @@
+# this workflow contains fmt,build,ut fot tikv-client
+name: verify
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: maven
+      - name: fmt
+        run: |
+          export LC_ALL=en_US.UTF-8
+          export LANG=en_US.UTF-8
+          export LANGUAGE=en_US.UTF-8
+          mvn mvn-scalafmt_2.12:format -Dscalafmt.skip=false -B
+          mvn com.coveo:fmt-maven-plugin:format -B
```

Comment:
Will scalafmt and javafmt conflict each other?

Reply:
They will be distincted by file extension name. For example, javafmt only scan the file end with .Java